Sources/ServiceModelGenerate/MockClientDelegate.swift

```diff
@@ -45,17 +45,20 @@ public struct MockClientDelegate: ModelClientDelegate {
         self.asyncAwaitGeneration = asyncAwaitGeneration
         
         let name: String
+        let additionalConformingProtocol: String
```

Comment:
I suppose if we added a third protocol we would rename this string, as "additional" wouldn't really make sense then?  Might be better to rename this now, but will leave this up to you.

Reply:
Yup, renamed.